### PR TITLE
Default colors refactoring

### PR DESCRIPTION
**Pull request by rurikoaraki**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Saw some duplicated code so deleting it and restructuring some files so that the duplication doesn't exist anymore. Specifically,  paletteFromFabricColors and getStockWebHCPalette are duplicated, while getStockWebPalette and getStockWebDarkPalette have some differences with the base version

### CODE REVIEW

File: packages/theming/default-theme/src/stockWebPalette.macos.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
```
Comment (rurikoaraki) [inline packages/theming/default-theme/src/stockWebPalette.macos.ts:2]: This file's contents are unchanged from defaultColors.macos.ts

File: packages/theming/default-theme/src/stockWebPalette.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
```
Comment (rurikoaraki) [inline packages/theming/default-theme/src/stockWebPalette.ts:2]: This file's contents are unchanged from defaultColors.ts

File: packages/theming/default-theme/src/stockWebPalette.win32.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+import type { ThemeColorDefinition } from '@fluentui-react-native/theme-types';
+
+import { createColorAliasTokens } from './createAliasTokens';
```
Comment (rurikoaraki) [inline packages/theming/default-theme/src/stockWebPalette.win32.ts:5]: This file's contents are unchanged from defaultColors.win32.ts

**Review by PPatBoyd — APPROVED** [review]

**Comment by Saadnajmi** [comment]
I'm a little confused, there are platform specific "stock web palettes? Granted if that's how it was before, I probably contributed to that.

**Comment by rurikoaraki** [comment]
> I'm a little confused, there are platform specific "stock web palettes? Granted if that's how it was before, I probably contributed to that.

Yeah I'm not entirely sure how this came to be, but it's to populate some older colors in the theme. These are remnants from v8 or so.

**Pull request merged**